### PR TITLE
Supports overlay video rendering

### PR DESCRIPTION
**Pull request by stefanalund**
Makes it possible for Hybrid OpenWebRTC apps, such as Bowser, to use
native OpenGL video rendering. Includes minor other fixes.

### CODE REVIEW

**Comment by stefanalund** [comment]
@pererikb can you review, push and also update the live app at demo.openwebrtc.org?

File: web/client/webrtc_example.html [hunk 1]
```diff
@@ -40,7 +40,7 @@
             <div id="log_div"></div>
         </section>
         <footer>
-            Copyright Ericsson Research 2014
+            Copyright Ericsson Research 2015
```
Comment (superdump) [inline web/client/webrtc_example.html]: 2016 :)

**Comment by superdump** [comment]
Needs rebasing?

**Comment by stefanalund** [comment]
@superdump done.

**Pull request closed without merging**